owncloudApp/src/main/java/com/owncloud/android/presentation/ui/files/filelist/MainFileListFragment.kt

```diff
@@ -564,6 +711,43 @@ class MainFileListFragment : Fragment(), SortDialogListener, SortOptionsView.Sor
         }
     }
 
+    fun syncFiles(files: Collection<OCFile?>) {
+        for (file in files) {
+            file?.let { syncFile(file) }
+        }
+    }
+
+    fun syncFile(file: OCFile) {
+        if (!file.isFolder) {
+            // TODO Sync file
+        } else {
+            filesViewModel.refreshFolder(file.remotePath)
+        }
+    }
+
+    fun getProgressBar(): ProgressBar {
+        return binding.syncProgressBar
+    }
+
+    fun getShadowView(): View {
+        return binding.shadowView
+    }
+
+    fun setMessageForEmptyList(message: String) {
+        binding.emptyDataParent.root.visibility = View.VISIBLE
+        binding.emptyDataParent.listEmptyDatasetIcon.visibility = View.GONE
+        binding.emptyDataParent.listEmptyDatasetTitle.visibility = View.GONE
+        binding.emptyDataParent.listEmptyDatasetSubTitle.text = message
```

Comment:
we can reduce this using kotlin's sintax: `binding.emptyDataParent.let{}`

Reply:
:+1